cli/create.go

```diff
@@ -17,6 +18,7 @@ import (
 	vc "github.com/kata-containers/runtime/virtcontainers"
 	vf "github.com/kata-containers/runtime/virtcontainers/factory"
 	"github.com/kata-containers/runtime/virtcontainers/pkg/oci"
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
I don't think you need to specify a custom name for the import here. `opentracing-go` will be automatically translated into `opentracing` to be used from the code.

Reply:
I didn't specify the name - it was added automatically by the go tooling.